pkg/identity/numericidentity.go

```diff
@@ -26,6 +26,7 @@ const (
 	// Identities also have scopes, which is defined by the high 8 bits.
 	// 0x00 -- global and reserved identities
```

Comment:
I have mild concerns with this, because reserved identities are not like global identities in a few key ways - they're not globally allocated and we generally avoid sending them on the wire. During upgrades if there are new reserved identities introduced, the older agents in the cluster will not know about them. I'm not sure if that has any bearing on this line of code but I don't think we should just categorize that upper 8 bits == 0 has the same properties. Those reserved identities with the prefix `0x000000xx` are special.

Reply:
right; this code doesn't change any behavior w.r.t reserved identities (with the obvious exception of remote-node). This is just documenting some already-existing expectations in the datapath.

If you come up with a better name for `IdentityScopeGlobal`, I'm happy to change.

Reply:
You're right; it's funny. Reserved identities are "global" in that all (up-to-date) have the same view of them; but they don't come from any centralized identity store. (Unless you consider quay.io centralized :-) ). That's why I called them out as such in the code comment.

Reply:
I think the middle ground here is to keep the current changes + a comment describing the brief difference for reserved identities.

Reply:
@nathanjsweet This is also unresolved ^

Reply:
@christarazi Why? I added details about the nature of reserved identities to the comment and specifically called out the version incompatibility problem.